Modules/rat_rescue.py

```diff
@@ -336,22 +336,37 @@ def board_index(self, value: int or None) -> None:
             raise TypeError(f"expected int or None, got {type(value)}")
 
     @property
-    def case_id(self) -> UUID:
+    def uuid(self) -> UUID:
         """
         The API Id of the rescue.
 
         Returns: API id
 
-        Notes:
-            This field is **READ ONLY**
         """
 
         return self._id
 
+    @uuid.setter
+    def uuid(self, uuid) -> None:
```

Comment:
Setter convention dictates the signature is
 ```py
def uuid(self, value)
```
Further, it may prove useful to put a type hint in here, eg
```py
def uuid(self, value:UUID)
```